src/hotspot/share/opto/macroArrayCopy.cpp

```diff
@@ -1266,7 +1266,7 @@ void PhaseMacroExpand::expand_arraycopy_node(ArrayCopyNode *ac) {
     generate_arraycopy(ac, alloc, &ctrl, merge_mem, &io,
                        adr_type, T_OBJECT,
                        src, src_offset, dest, dest_offset, length,
-                       true, !ac->is_copyofrange());
```

Comment:
We cannot use this anymore, since we then ignore the `has_negative_length_guard` for `copyOfRange` (and generate the negative length guard twice).